models/airdrop/apecoin/apecoin_airdrop_transfers.sql

```diff
@@ -0,0 +1,132 @@
+{{ config(
+    alias = 'apecoin_airdrop_transfers',
+    partition_by = ['transfer_block_date'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['transfer_tx_hash', 'transfer_evt_index', 'transfer_block_number']
+    )
+}}
+
+-- https://dune.com/queries/511739 h/t hildobby
+{% set airdrop_start_date = '2022-03-16' %}
+{% set airdrop_end_date = '2022-03-19' %}
+
+SELECT
```

Comment:
definitely nitpicking: missing indentation

Reply:
where should the indentation be? The start of the line?